CollapseLauncher/Classes/RepairManagement/Genshin/Check.cs

```diff
@@ -51,7 +52,9 @@ private async Task Check(List<PkgVersionProperties> assetIndex, CancellationToke
             }
             catch (AggregateException ex)
             {
-                throw ex.Flatten().InnerExceptions.First();
+                var innerExceptionsFirst = ex.Flatten().InnerExceptions.First();
+                SentryHelper.ExceptionHandler(innerExceptionsFirst, SentryHelper.ExceptionType.UnhandledOther);
+                throw innerExceptionsFirst;
```

Comment:
I don't think throwing ``AggregateException`` into Sentry will be a good idea since it can actually be another type exception like ``OperationCancelledException`` or ``TaskCancelledException``. In my honest opinion, I think we need to check the ``innerExceptionsFirst`` first to check if it's no other than ``OperationCancelledException`` or ``TaskCancelledException``, then just throw it. Other than that, just ignore it